qa/rpc-tests/mempool_size_limit_more.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python3
```

Comment:
Throughout the test, when dealing with transactions, there's a mix of the terms fee and fee rate (`minfee`, `maxfee`, `feerate`); it is better to make this more coherent using only fee rate (the quantity being actually used for transactions ordering).

Reply:
Thanks, I rephrased all the log messages and comments along the test to avoid confusion.